<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite skipped="1" hostname="EQ-EQ6283668" name="org.TestcaseAPI.TC_Get_Location" tests="2" failures="0" timestamp="18 Jul 2019 17:54:34 GMT" time="0.039" errors="1">
  <testcase name="TC1_Maps" time="0.000" classname="org.TestcaseAPI.TC_Get_Location">
    <skipped/>
  </testcase> <!-- TC1_Maps -->
  <testcase name="beforeSetup" time="0.039" classname="org.TestcaseAPI.TC_Get_Location">
    <error type="java.io.FileNotFoundException" message="C:\Users\WZJS1251\eclipse-workspace\ApiAutomation\src\main\java\org\config\Configurations.properties (The system cannot find the path specified)">
      <![CDATA[java.io.FileNotFoundException: C:\Users\WZJS1251\eclipse-workspace\ApiAutomation\src\main\java\org\config\Configurations.properties (The system cannot find the path specified)
at java.io.FileInputStream.open0(Native Method)
at java.io.FileInputStream.open(FileInputStream.java:195)
at java.io.FileInputStream.<init>(FileInputStream.java:138)
at java.io.FileInputStream.<init>(FileInputStream.java:93)
at org.BaseSetup.IntialiseBase.beforeSetup(IntialiseBase.java:35)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:59)
at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:455)
at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:222)
at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:142)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:401)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </error>
  </testcase> <!-- beforeSetup -->
</testsuite> <!-- org.TestcaseAPI.TC_Get_Location -->
